config/connections.js

```diff
@@ -90,9 +90,9 @@ module.exports.connections = {
   ***************************************************************************/
 
   // Store data in memory. Used for tests
-  memory: {
-    adapter: 'sails-disk',
-    inMemory: true
-  }
+  // memory: {
+    // adapter: 'sails-disk',
+    // inMemory: true
+  // }
 
 };
```

Comment:
This `connections.js` file needs to be refactored a bit. I think all these changes can be reverted.